itou/templates/signup/prescriber_siren.html

```diff
@@ -0,0 +1,96 @@
+{% extends "layout/content_small.html" %}
+{% load bootstrap4 %}
+{% load static %}
+
+{% block title %}Prescripteur/Orienteur - Inscription{{ block.super }}{% endblock %}
+
+{% block content %}
+
+    <h1>
+        Inscription
+        <small class="text-muted">Prescripteur/Orienteur</small>
+    </h1>
+
+    <div class="alert alert-secondary" role="alert">
+        Retrouvez facilement votre numéro SIREN à partir du nom de votre organisation sur le site
+        <a href="https://sirene.fr/" rel="noopener" target="_blank">
+            sirene.fr
+            {% include "includes/icon.html" with icon="external-link" %}
+        </a>
+    </div>
+
+    <form method="post" action="" class="js-prevent-multiple-submit">
+
+        {% csrf_token %}
+
+        {% bootstrap_form form %}
```

Comment:
```suggestion
        {% bootstrap_form form alert_error_type="all" %}
```

Sinon j'en connais un [qui ne sera pas content](https://github.com/betagouv/itou/pull/768/files) 😨